### PR TITLE
State

### PR DESCRIPTION
**Pull request by Willieodwyer**

### CODE REVIEW

File: GameFrame/SpeedState/SpeedMud.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class SpeedMud : ISpeedState
 
         public SpeedMud()
         {
```
Comment (Taikatou) [inline GameFrame/SpeedState/SpeedMud.cs]: Can we remove SpeedState. Do not think it should be hard baked into framework

Comment (Willieodwyer) [inline GameFrame/SpeedState/SpeedMud.cs]: no you need a concrete state

Comment (Taikatou) [inline GameFrame/SpeedState/SpeedMud.cs]: Yes in demo

Comment (Taikatou) [inline GameFrame/SpeedState/SpeedMud.cs]: Not framework

Comment (Willieodwyer) [inline GameFrame/SpeedState/SpeedMud.cs]: that's not how state pattern werks

**Pull request merged**